samples/sensor/adc_cmp_npcx/sample.yaml

```diff
@@ -0,0 +1,14 @@
+sample:
+  description: Demonstration of nuvoton NPCX ADC comparator driver
+  name: NPCX ADC comparator sample
+common:
+  platform_allow: npcx9m6f_evb
+  integration_platforms:
+    - npcx9m6f_evb
+  tags: drivers
+tests:
+  sample.sensor.adc_cmp:
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
```

Comment:
Aren't you missing a `regex` here? What is the test acceptance criteria?

Reply:
Done.